packages/lavapack/src/runtime.js

```diff
@@ -70,6 +70,17 @@
 })({ onStatsReady }) : () => {}
   const createKernel = // LavaMoat Prelude
 (function () {
+  const globalRef = globalThis;
+
+  const {
+    includeSnow,
+  } = {"includeSnow":false}
+
+  const SNOW = globalRef.SNOW;
+  if (!SNOW && includeSnow) {
+    throw 'LavaMoat - Snow is expected to exist but it does not'
```

Comment:
```suggestion
    throw new Error('LavaMoat - Snow is expected to exist but it does not');
```
?
Or if there's a motivation behind throwing a string, an explanatory comment would be great